pkg/services/dns/dns_config_unix.go

```diff
@@ -3,16 +3,23 @@
 package dns
 
 import (
+	"net"
+
 	"github.com/miekg/dns"
 )
 
-func getDNSHostAndPort() (string, string, error) {
+func getDNSHostAndPort() ([]string, error) {
 	conf, err := dns.ClientConfigFromFile("/etc/resolv.conf")
 	if err != nil {
-		return "", "", err
+		return []string{}, err
 	}
-	// TODO: use all configured nameservers, instead just first one
-	nameserver := conf.Servers[0]
-
-	return nameserver, conf.Port, nil
+	var hosts = make([]string, len(conf.Servers))
+	for _, server := range conf.Servers {
+		dnsIP := net.ParseIP(server)
+		// add only ipv4 dns addresses
+		if dnsIP != nil && dnsIP.To4() != nil {
```

Comment:
You could also use `net/netip` and  `ParseAddr` + `Is4`, the code would be more similar to what is done for Windows. I'm fine with the current code if you prefer to keep this way.